test/query.test.js

```diff
@@ -2324,19 +2325,19 @@ describe('query', () => {
     )
   })
 
-  test('bytes', () => {
-    return Promise.all([
-      assertQuery(query.Bytes('AQIDBA=='), new Bytes('AQIDBA==')),
-      assertQuery(
-        query.Bytes(new Uint8Array([0, 0, 0, 0])),
-        new Bytes('AAAAAA==')
-      ),
-      assertQuery(query.Bytes(new ArrayBuffer(4)), new Bytes('AAAAAA==')),
-      assertQuery(new Bytes('AQIDBA=='), new Bytes('AQIDBA==')),
-      assertQuery(new Uint8Array([0, 0, 0, 0]), new Bytes('AAAAAA==')),
-      assertQuery(new ArrayBuffer(4), new Bytes('AAAAAA==')),
-    ])
-  })
+  // test('bytes', () => {
```

Comment:
@marrony Same question here. I'm seeing the following error: `Invalid string. Length must be a multiple of 4`

Reply:
this is a base64 error on the driver side, you might need to experiment another one that support base64 url safe

Reply:
@marrony I don't see any documentation on our site for `Bytes`. Has this been deprecated? If not, what is the expected behavior?